.gitignore

```diff
@@ -5,6 +5,8 @@
 *.so
 *.o
 
+*#
```

Comment:
What generates these?

Reply:
Emacs generates `#*#` files either as back files when editing or when not saving changes when exiting.  I should have annotated that in the `.gitignore`